src/common/database.c

```diff
@@ -2524,6 +2533,122 @@ gboolean dt_database_get_lock_acquired(const dt_database_t *db)
   return db->lock_acquired;
 }
 
+void _dt_database_vacuum(const struct dt_database_t *db)
+{
+  sqlite3_exec(db->handle, "VACUUM data", NULL, NULL, NULL);
+  sqlite3_exec(db->handle, "VACUUM data", NULL, NULL, NULL);
+  sqlite3_exec(db->handle, "ANALYZE main", NULL, NULL, NULL);
+  sqlite3_exec(db->handle, "ANALYZE main", NULL, NULL, NULL);
+}
+
+gboolean _ask_for_vacuum(const gboolean has_gui, guint64 size)
```

Comment:
same here, vacuum not good name should be sync with new name as discussed above.